app/assets/javascripts/models/beta/label.js

```diff
@@ -31,3 +31,5 @@ export const joinLabels = (labels) =>
 
 export const getNames = (labels) =>
   labels.map(label => label.name);
+
+export const hasLabel = (labels, label) => labels.includes(label);
```

Comment:
Maybe we should do it by the `name` instead of using object reference?

Reply:
Actually looks like it's not being used anymore.

Reply:
Yes, I will remove this function.